### PR TITLE
feat(icon): bacteria, DNA and Virus icons added to the library

### PR DESCRIPTION
**Pull request by masoudmanson**
## Summary

**Icon**
Shortcut ticket: [sh-214394](https://app.shortcut.com/sci-design-system/story/214394/icon-component)

Component Requirements: Addition of the Bacteria Icon, DNA, Virus

https://drive.google.com/file/d/1-CdIBGB9C4RxdpjYKLMaOGZPmYavwSYf/view
https://drive.google.com/file/d/1-C3lPIJtdVDc4EPdH4t1lYrDn-P5oq7D/view
https://drive.google.com/file/d/1-FkGfps4FppPj_6saYVn57UdXU5SaQtI/view

Team(s) Requesting: CZ ID
Component Needed By: September 23rd

### CODE REVIEW

File: src/core/Icon/map.ts [hunk 1]
```diff
@@ -77,11 +79,13 @@ import { ReactComponent as IconTreeHorizontalPrivateLarge } from "../../common/s
 import { ReactComponent as IconTreeHorizontalPublicLarge } from "../../common/svgs/IconTreeHorizontalPublicLarge.svg";
 import { ReactComponent as IconTreeHorizontalSmall } from "../../common/svgs/IconTreeHorizontalSmall.svg";
 import { ReactComponent as IconTreeVerticalSmall } from "../../common/svgs/IconTreeVerticalSmall.svg";
+import { ReactComponent as IconVirusLarge } from "../../common/svgs/IconVirusLarge.svg";
 import { ReactComponent as IconXMarkCircleSmall } from "../../common/svgs/IconXMarkCircleSmall.svg";
 import { ReactComponent as IconXMarkLarge } from "../../common/svgs/IconXMarkLarge.svg";
 import { ReactComponent as IconXMarkSmall } from "../../common/svgs/IconXMarkSmall.svg";
 
 export interface IconNameToSizes {
+  bacteria: "l";
```
Comment (mmmmmmaya) [inline src/core/Icon/map.ts:88]: typically we allow `l` and `xl` for an icon with a large image. Any reason why that isn't the case here?

File: src/core/Icon/index.stories.tsx [hunk 2]
```diff
@@ -66,41 +73,138 @@ IconInteractive.args = {
...
-      }}
-    >
+    <IconBankWrapper>
       {icons.map(([sdsIcon, icon]) => {
         const { largeIcon } = icon;
         const sdsSize = largeIcon ? "l" : "s";
 
+        const [copied, setCopied] = useState(false);
+
+        const copyIconNameHandler = (iconName: string) => {
```
Comment (mmmmmmaya) [inline src/core/Icon/index.stories.tsx:174]: cool idea

**Review by mmmmmmaya — APPROVED** [review]

File: src/core/Icon/map.ts (hunk 1 above)
Comment (masoudmanson) [reply src/core/Icon/map.ts:88]: Since the image is titled "Large," I thought there should only be a large size. I'll now add the xl sizes. 🥸

**Comment by ovalenzuela19** [comment]
Thanks so much for adding these icons in so quickly!!

**Pull request merged**